### ISSUE TITLE
font size for image export

### ISSUE THREAD

**Issue by dave256** [Kamik423/equalize#15]
I realize there are some options in LaTeX for specifying the font size, but it would be nice if you could have a setting for a larger font size so you don’t need to type those for every image export. I have a whiteboard app I use (I’m also the developer of it) - reDraw Whiteboard that I just purchased Equalize for use with so I can drag math equations from Equalize onto the whiteboard canvas. I can also drag the PDFs (I purchase the Pro unlock) which seem to be vectors, but I’d prefer to drag images (of a larger size).

**Comment by Kamik423** [comment]
Are you aware you can pinch to zoom to get different font sizes?

**Comment by dave256** [comment]
Ah, I was not. Is there a reason  you stopped at 150 pt? You can close the issue but maybe allowing it to go up a little higher would be nice. Somewhere between 100 and 150 is what I would probably use but even a little larger might be nice at times. Thanks!

**Comment by Kamik423** [comment]
You might be able to increase the image size in the whiteboard app too. I capped the size as I thought 150 would be enough for anyone and I have to cap it as not to deal with bugs that happen when people scale the image up too big. How much do you require font sizes above 150?

**Comment by dave256** [comment]
Yes, I can scale it in my whiteboard app also but then I may get some aliasing issues. If it causes bugs above when you scale above 150, don't worry about it. Thanks for the follow-up.

**Comment by Kamik423** [comment]
150 isn't a hard limit so I might change it at some point, but there needs to be some upper limit and 150 seemed reasonable at the time.
